docs/configure/images-and-assets.md

```diff
@@ -137,3 +135,18 @@ In this case, you need to have all your images and media files with relative pat
 If you load static content via importing, this is automatic, and you do not have to do anything.
 
 Suppose you are serving assets in a [static directory](#serving-static-files-via-storybook-configuration) along with your Storybook. In that case, you need to use relative paths to load images or use the base element.
+
+Example:
```

Comment:
This line can be axed; I felt compelled to tie the idea of static-imports having relative pathing to above the section below, but I can see how this is now like a dangling line.